ads/mediaimpact.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript} from '../src/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+
+export function mediaimpact(global, data) {
+  const src = "https://ec-ns.sascdn.com/diff/251/bild.js";
+  writeScript(global, src);
+  /*global.SmartAdServerAjax = function() {
```

Comment:
Kind of. We, MediaImpact, use SmartAdServer for delivery but we have our own lib to generate the adcalls. For historical reasons we have to use the same function-calls but treat them separatly to SmartAdServer in general.

So it's related but completly different.